contribs/gmf/src/editing/editFeatureComponent.js

```diff
@@ -563,6 +563,12 @@ Controller.prototype.$onInit = function () {
     throw new Error('Missing map');
   }
 
+  this.options_.highlightStyle.circle = {
+    radius: 7,
+    fill: this.options_.highlightStyle.fill,
+    stroke: this.options_.highlightStyle.stroke,
+  };
```

Comment:
Why this?
Looks like that it should be moved here: https://github.com/camptocamp/c2cgeoportal/blob/2.6/geoportal/c2cgeoportal_geoportal/scaffolds/update/geoportal/CONST_vars.yaml_tmpl#L319